### PR TITLE
Add Aiven Kubernetes operator page

### PR DESCRIPTION
**Pull request by jonatasbaldin**
Hi there 👋 

This PR adds the "Aiven Kubernetes operator" page. It's flag as a draft since the Operator is not publicly released yet.

### CODE REVIEW

File: docs/tools/kubernetes-operator.rst [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+Install the operator
+''''''''''''''''''''
+Once you have a Kubernetes cluster and an Aiven authentication token, we can proceed to install the Operator.
+
+Install the cert-manager with the command below. It is used to manage the webhook TLS certificates used by our Operator.
+
+.. code:: bash
+
+    $ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.5.3/cert-manager.yaml
```
[the remark this answers is not in the document]
Comment (jonatasbaldin) [reply docs/tools/kubernetes-operator.rst]: Done!

File: .github/styles/Aiven/capitalization_headings.yml [hunk 2]
```diff
@@ -6,3 +6,4 @@ scope: heading
 match: $sentence
 exceptions:
   - HowTo
+  - Kubernetes
```
[the remark this answers is not in the document]
Comment (jonatasbaldin) [reply .github/styles/Aiven/capitalization_headings.yml]: Done!

**Comment by jonatasbaldin** [comment]
I also added a new twist to the Aiven tools CLI index page.

File: docs/tools/aiven-operator.rst [hunk 3]
```diff
@@ -0,0 +1,217 @@
+Aiven Operator for Kubernetes
+=============================
+
+    Kubernetes, also known as K8s, is an open-source system for automating deployment, scaling, and management of containerized applications. Kubernetes Operators are software extensions to Kubernetes that make use of custom resources to manage applications and their components. `Kubernetes website <https://kubernetes.io/>`_.
```
Comment (lornajane) [inline docs/tools/aiven-operator.rst:4]: here (and elsewhere), we have "Operator" with a capital O, but it isn't a proper noun other than in the title of the project "Aiven Operator". The same goes for other English words here, such as Pod, Secret, etc. They do have special meanings in Kubernetes , but writing them in an unexpected way doesn't make the text easier to read.

File: docs/tools/aiven-operator.rst [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+.. code:: bash
+
+    kubectl get pod -l app.kubernetes.io/name=aiven-operator
+
+    NAME                              READY   STATUS    RESTARTS   AGE
+    aiven-operator-576d944499-ggttj   1/1     Running   0          12m
+
+Authenticating
+''''''''''''''
+Before creating a service, we need to authenticate the Operator with Aiven's API. To do so, create the Kubernetes Secret with the command below, substituting the "<your-token-here>" with the authentication token generated in the prerequisites section.
```
Comment (lornajane) [inline docs/tools/aiven-operator.rst:74]: fixed-width for placeholders

**Review by lornajane — CHANGES_REQUESTED** [review]
Some first-impressions feedback,I'll take another look when I can

File: .github/styles/Aiven/capitalization_headings.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (jonatasbaldin) [reply .github/styles/Aiven/capitalization_headings.yml]: Fixed.

File: docs/tools/aiven-operator.rst (hunk 3 above)
Comment (jonatasbaldin) [reply docs/tools/aiven-operator.rst:4]: Fixed.

Just stating here: IMHO, Kubernetes-specific resources (like Pods and Secrets) should be capitalized when talking about Kubernetes heheh

File: docs/tools/aiven-operator.rst (hunk 4 above)
Comment (jonatasbaldin) [reply docs/tools/aiven-operator.rst]: Fixed.

**Review by lornajane — APPROVED** [review]
This is very nicely done! It is well-structured, the example is simple but gives great context to be useful. I especially liked the call to get involved with the project since it is open source as well ....  Thanks @jonatasbaldin !

**Comment by jonatasbaldin** [comment]
Let's :shipit:

**Pull request merged**